diff --git a/8Knot/app.py b/8Knot/app.py
index 611fd26..7c17d47 100644
--- a/8Knot/app.py
+++ b/8Knot/app.py
@@ -83,6 +83,11 @@ app = dash.Dash(
 server = app.server
 server = _login.configure_server_login(server)
 
+from werkzeug.middleware.proxy_fix import ProxyFix
+
+server.wsgi_app = ProxyFix(
+    server.wsgi_app, x_for=1, x_proto=1, x_host=1, x_prefix=1
+)
 
 """DASH PAGES LAYOUT"""
 # layout of the app stored in the app_layout file, must be imported after the app is initiated