diff --git a/8Knot/queries/affiliation_query.py b/8Knot/queries/affiliation_query.py
index db7d2df..dad1cd4 100644
--- a/8Knot/queries/affiliation_query.py
+++ b/8Knot/queries/affiliation_query.py
@@ -35,7 +35,7 @@ def affiliation_query(self, repos):
 
     query_string = f"""
                     SELECT
-                        left(ca.cntrb_id::text, 15) as cntrb_id, -- first 15 characters of the uuid
+                        left(c.cntrb_id::text, 15) as cntrb_id, -- first 15 characters of the uuid
                         timezone('utc', c.created_at) AS created_at,
                         c.repo_id,
                         c.login,
