diff --git a/8Knot/pages/chaoss/visualizations/contrib_importance_pie.py b/8Knot/pages/chaoss/visualizations/contrib_importance_pie.py
index 6f53daf..15b6d31 100644
--- a/8Knot/pages/chaoss/visualizations/contrib_importance_pie.py
+++ b/8Knot/pages/chaoss/visualizations/contrib_importance_pie.py
@@ -42,7 +42,6 @@ gc_contrib_importance_pie = VisualizationAIO(
             dcc.Dropdown(
                 id=f"action-type-{PAGE}-{VIZ_ID}",
                 options=[
-                    {"label": "Commit", "value": "Commit"},
                     {
                         "label": "Issue Opened",
                         "value": "Issue Opened",
@@ -68,7 +67,7 @@ gc_contrib_importance_pie = VisualizationAIO(
                         "value": "PR Comment",
                     },
                 ],
-                value="Commit",
+                value="PR Review",
                 clearable=False,
                 className="dark-dropdown",
             ),
diff --git a/8Knot/pages/chaoss/visualizations/project_velocity.py b/8Knot/pages/chaoss/visualizations/project_velocity.py
index 24d0c7d..96151f8 100644
--- a/8Knot/pages/chaoss/visualizations/project_velocity.py
+++ b/8Knot/pages/chaoss/visualizations/project_velocity.py
@@ -266,8 +266,8 @@ def process_data(
     # replace all nan to 0
     df_consolidated.fillna(value=0, inplace=True)
 
-    # log of commits and contribs if values are not 0
-    df_consolidated["log_num_commits"] = df_consolidated["Commit"].apply(lambda x: math.log(x) if x != 0 else 0)
+    # log of pr_open and contribs if values are not 0
+    df_consolidated["log_num_pr_open"] = df_consolidated["PR Opened"].apply(lambda x: math.log(x) if x != 0 else 0)
     df_consolidated["log_num_contrib"] = df_consolidated["num_unique_contributors"].apply(
         lambda x: math.log(x) if x != 0 else 0
     )
@@ -300,13 +300,12 @@ def create_figure(df: pd.DataFrame, log):
     # graph generation
     fig = px.scatter(
         df,
-        x="log_num_commits",
+        x="log_num_pr_open",
         y=y_axis,
         color="repo_name",
         size="log_num_contrib",
         hover_data=[
             "repo_name",
-            "Commit",
             "PR Opened",
             "Issue Opened",
             "num_unique_contributors",
@@ -315,13 +314,13 @@ def create_figure(df: pd.DataFrame, log):
     )
 
     fig.update_traces(
-        hovertemplate="Repo: %{customdata[0]} <br>Commits: %{customdata[1]} <br>Total PRs: %{customdata[2]}"
-        + "<br>Total Issues: %{customdata[3]} <br>Total Contributors: %{customdata[4]}<br><extra></extra>",
+        hovertemplate="Repo: %{customdata[0]}  <br>Total PRs: %{customdata[2]} <br>Total Issues: %{customdata[3]} "
+        + "<br>Total Contributors: %{customdata[4]}<br><extra></extra>",
     )
 
     # layout styling
     fig.update_layout(
-        xaxis_title="Logarithmic Commits",
+        xaxis_title="Logarithmic PRs Opened",
         yaxis_title=y_title,
         margin_b=40,
         font=dict(size=14),
diff --git a/8Knot/pages/contributors/visualizations/contrib_importance_over_time.py b/8Knot/pages/contributors/visualizations/contrib_importance_over_time.py
index 857e1bf..f0001bd 100644
--- a/8Knot/pages/contributors/visualizations/contrib_importance_over_time.py
+++ b/8Knot/pages/contributors/visualizations/contrib_importance_over_time.py
@@ -207,7 +207,6 @@ def process_data(df, threshold, window_width, step_size):
 
     # dynamically calculate the contributor prolificacy over time for each of the action times and store results in df_final
     (
-        df_final["Commit"],
         df_final["Issue Opened"],
         df_final["Issue Comment"],
         df_final["Issue Closed"],
@@ -247,21 +246,11 @@ def create_figure(df_final, threshold, step_size):
     # create plotly express line graph
     fig = go.Figure(
         [
-            go.Scatter(
-                name="Commit",
-                x=df_final["period_from"],
-                y=df_final["Commit"],
-                text=action_types[0],
-                customdata=customdata,
-                mode="lines",
-                showlegend=True,
-                marker=dict(color=baby_blue[0]),
-            ),
             go.Scatter(
                 name="Issue Opened",
                 x=df_final["period_from"],
                 y=df_final["Issue Opened"],
-                text=action_types[1],
+                text=action_types[0],
                 customdata=customdata,
                 mode="lines",
                 showlegend=True,
@@ -271,7 +260,7 @@ def create_figure(df_final, threshold, step_size):
                 name="Issue Comment",
                 x=df_final["period_from"],
                 y=df_final["Issue Comment"],
-                text=action_types[2],
+                text=action_types[1],
                 customdata=customdata,
                 mode="lines",
                 showlegend=True,
@@ -281,7 +270,7 @@ def create_figure(df_final, threshold, step_size):
                 name="Issue Closed",
                 x=df_final["period_from"],
                 y=df_final["Issue Closed"],
-                text=action_types[3],
+                text=action_types[2],
                 customdata=customdata,
                 mode="lines",
                 showlegend=True,
@@ -291,7 +280,7 @@ def create_figure(df_final, threshold, step_size):
                 name="PR Opened",
                 x=df_final["period_from"],
                 y=df_final["PR Opened"],
-                text=action_types[4],
+                text=action_types[3],
                 customdata=customdata,
                 mode="lines",
                 showlegend=True,
@@ -301,7 +290,7 @@ def create_figure(df_final, threshold, step_size):
                 name="PR Comment",
                 x=df_final["period_from"],
                 y=df_final["PR Comment"],
-                text=action_types[5],
+                text=action_types[4],
                 customdata=customdata,
                 mode="lines",
                 showlegend=True,
@@ -311,7 +300,7 @@ def create_figure(df_final, threshold, step_size):
                 name="PR Review",
                 x=df_final["period_from"],
                 y=df_final["PR Review"],
-                text=action_types[6],
+                text=action_types[5],
                 customdata=customdata,
                 mode="lines",
                 showlegend=True,
@@ -359,8 +348,7 @@ def cntrb_prolificacy_over_time(df, period_from, period_to, window_width, thresh
     df_in_range = df.loc[time_mask]
 
     # initialize varibles to store contributor prolificacy accoding to action type
-    commit, issueOpened, issueComment, issueClosed, prOpened, prReview, prComment = (
-        None,
+    issueOpened, issueComment, issueClosed, prOpened, prReview, prComment = (
         None,
         None,
         None,
@@ -376,7 +364,6 @@ def cntrb_prolificacy_over_time(df, period_from, period_to, window_width, thresh
     # pivot df such that the column names correspond to the different action types, index is the cntrb_ids, and the values are the number of contributions of each contributor
     df_count_cntrbs = df_count_cntrbs.pivot(index="cntrb_id", columns="Action", values="count")
 
-    commit = calc_lottery_factor(df_count_cntrbs, "Commit", threshold)
     issueOpened = calc_lottery_factor(df_count_cntrbs, "Issue Opened", threshold)
     issueComment = calc_lottery_factor(df_count_cntrbs, "Issue Comment", threshold)
     issueClosed = calc_lottery_factor(df_count_cntrbs, "Issue Closed", threshold)
@@ -384,7 +371,7 @@ def cntrb_prolificacy_over_time(df, period_from, period_to, window_width, thresh
     prReview = calc_lottery_factor(df_count_cntrbs, "PR Review", threshold)
     prComment = calc_lottery_factor(df_count_cntrbs, "PR Comment", threshold)
 
-    return commit, issueOpened, issueComment, issueClosed, prOpened, prReview, prComment
+    return issueOpened, issueComment, issueClosed, prOpened, prReview, prComment
 
 
 def calc_lottery_factor(df, action_type, threshold):
diff --git a/8Knot/pages/contributors/visualizations/contrib_importance_pie.py b/8Knot/pages/contributors/visualizations/contrib_importance_pie.py
index a213621..48f7557 100644
--- a/8Knot/pages/contributors/visualizations/contrib_importance_pie.py
+++ b/8Knot/pages/contributors/visualizations/contrib_importance_pie.py
@@ -44,7 +44,6 @@ gc_contrib_importance_pie = VisualizationAIO(
                     dcc.Dropdown(
                         id=f"action-type-{PAGE}-{VIZ_ID}",
                         options=[
-                            {"label": "Commit", "value": "Commit"},
                             {
                                 "label": "Issue Opened",
                                 "value": "Issue Opened",
@@ -70,7 +69,7 @@ gc_contrib_importance_pie = VisualizationAIO(
                                 "value": "PR Comment",
                             },
                         ],
-                        value="Commit",
+                        value="PR Review",
                         clearable=False,
                         className="dark-dropdown",
                     ),
