diff --git a/8Knot/cache_manager/db_init.py b/8Knot/cache_manager/db_init.py
index fc1688b..531f2d9 100644
--- a/8Knot/cache_manager/db_init.py
+++ b/8Knot/cache_manager/db_init.py
@@ -204,7 +204,6 @@ def _create_application_tables() -> None:
                 repo_id int,
                 login text,
                 action text,
-                rank int,
                 cntrb_company text,
                 email_list text
             )
@@ -220,8 +219,7 @@ def _create_application_tables() -> None:
                 cntrb_id text,
                 created_at text,
                 login text,
-                action text,
-                rank int
+                action text
             )
             """
         )
diff --git a/8Knot/pages/codebase/visualizations/cntrb_file_heatmap.py b/8Knot/pages/codebase/visualizations/cntrb_file_heatmap.py
index 041fe37..5c6229d 100644
--- a/8Knot/pages/codebase/visualizations/cntrb_file_heatmap.py
+++ b/8Knot/pages/codebase/visualizations/cntrb_file_heatmap.py
@@ -432,7 +432,7 @@ def cntrb_to_last_activity(df_actions: pd.DataFrame, df_dynamic_directory: pd.Da
     # drop unneccessary columns not needed after preprocessing steps
     df_actions = df_actions.reset_index()
     df_actions.drop(
-        ["index", "repo_id", "repo_name", "login", "Action", "rank"],
+        ["index", "repo_id", "repo_name", "login", "Action"],
         axis=1,
         inplace=True,
     )
diff --git a/8Knot/pages/codebase/visualizations/reviewer_file_heatmap.py b/8Knot/pages/codebase/visualizations/reviewer_file_heatmap.py
index c8c80aa..23f5196 100644
--- a/8Knot/pages/codebase/visualizations/reviewer_file_heatmap.py
+++ b/8Knot/pages/codebase/visualizations/reviewer_file_heatmap.py
@@ -431,7 +431,7 @@ def cntrb_to_last_activity(df_actions: pd.DataFrame, df_dynamic_directory: pd.Da
     # drop unneccessary columns not needed after preprocessing steps
     df_actions = df_actions.reset_index()
     df_actions.drop(
-        ["index", "repo_id", "repo_name", "login", "Action", "rank"],
+        ["index", "repo_id", "repo_name", "login", "Action"],
         axis=1,
         inplace=True,
     )
diff --git a/8Knot/queries/affiliation_query.py b/8Knot/queries/affiliation_query.py
index a35d413..db7d2df 100644
--- a/8Knot/queries/affiliation_query.py
+++ b/8Knot/queries/affiliation_query.py
@@ -35,12 +35,11 @@ def affiliation_query(self, repos):
 
     query_string = f"""
                     SELECT
-                        left(c.cntrb_id::text, 15), -- first 15 characters of the uuid
+                        left(ca.cntrb_id::text, 15) as cntrb_id, -- first 15 characters of the uuid
                         timezone('utc', c.created_at) AS created_at,
                         c.repo_id,
                         c.login,
                         c.action,
-                        c.rank,
                         con.cntrb_company,
                         array_to_string(
                             ARRAY(
@@ -62,7 +61,7 @@ def affiliation_query(self, repos):
                         c.repo_id in %s
                         and timezone('utc', c.created_at) < now() -- created_at is a timestamptz value
                         -- don't need to check non-null for created_at because it's non-null by definition.
-                    GROUP BY c.cntrb_id, c.created_at, c.repo_id, c.login, c.action, c.rank, con.cntrb_company, con.cntrb_email, con.cntrb_canonical
+                    GROUP BY c.cntrb_id, c.created_at, c.repo_id, c.login, c.action, con.cntrb_company, con.cntrb_email, con.cntrb_canonical
                     ORDER BY
                         c.created_at
                     """
diff --git a/8Knot/queries/contributors_query.py b/8Knot/queries/contributors_query.py
index ff67812..c6e46bf 100644
--- a/8Knot/queries/contributors_query.py
+++ b/8Knot/queries/contributors_query.py
@@ -44,8 +44,7 @@ def contributors_query(self, repos):
                         left(ca.cntrb_id::text, 15) as cntrb_id, -- first 15 characters of the uuid
                         timezone('utc', ca.created_at) AS created_at,
                         ca.login,
-                        ca.action,
-                        ca.rank
+                        ca.action
                     FROM
                         explorer_contributor_actions ca
                     WHERE
